iup/__init__.py

```diff
@@ -199,7 +168,7 @@ def trim_outlier_intervals(
           threshold (float): maximum standardized interval between first two dates
 
         Returns
-        IncidentUptakeData
+        IncidentValidatedUptake
```

Comment:
"IncidentValidatedUptake" should be "IncidentUptakeData" - why has the reverse substitution been made? The latter is defined as a class and the former is not.

Reply:
This is an error in replacing, it should be `IncidentUptakeData`!